src/components/Calendar.tsx

```diff
@@ -29,13 +29,20 @@ const { DOT_SIZE, MAX_DOTS_PER_DAY } = CALENDAR;
 type Props = {
   isListRefreshing: boolean;
   query: string;
-  queryVariables?: { dateRange?: string[]; contentContainerId?: number };
+  queryVariables: { dateRange?: string[]; contentContainerId?: number };
   navigation: StackNavigationProp<any>;
+  additionalData?: any;
 };
 
 const today = moment().format('YYYY-MM-DD');
 
-export const Calendar = ({ isListRefreshing, query, queryVariables, navigation }: Props) => {
+export const Calendar = ({
+  isListRefreshing,
+  query,
+  queryVariables,
+  navigation,
+  additionalData
```

Comment:
Shouldn't `additionalData` be included in the `markedDates` function? At the moment I can't see the dots in Volunteer's calendar, and I can't click on any day in the calendar because there are no dots.

Reply:
f602ddd

<img width="372" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/9f6ca9c7-e4dc-48ec-ae5a-d21911f3019d">